models/BaseEntity.cfc

```diff
@@ -444,7 +452,13 @@ component accessors="true" {
 			if ( value.virtual && !withVirtualAttributes ) {
 				return items;
 			}
-			items.append( asColumnNames ? value.column : key );
+			items.append( 
+				asColumnNames 
+				? value.isParentColumn 
+					? ( variables._parentEntity.meta.table & "." & value.column ) 
```

Comment:
Is `_parentEntity` an actual entity instance?  If so, then:
```suggestion
					? variables._parentEntity.qualifyColumn( value.column ) 
```

Reply:
No `_parentEntity` is the parent's definition with a nested `_meta` key.